### DISCUSSION TITLE
Line Type.... Hide/Show

### DISCUSSION THREAD

**Discussion by djannihilator** [OrcaSlicer/OrcaSlicer#6021] [Q&A]
Total newbie to Orca Slicer and still newbie to 3D printing.
I have learned how to find the super tiny ultra small tabs that are used to show/hide the prepare and preview settings and I can hide/show the color scheme.....
But after I hit the preview button, how in the hell do I hide the "line type" settings?

I want full screen to work on the model without the model ever hiding behind any of the gcodes.

Is there a super tiny button that I am missing (That I can't see)?

Is there a website/page with all of the hotkey commands?

Sincerely,
Raul.

**Comment by foreachthing** [comment]
There, press this hellish button and get the hell some more screen-space so you can see the hell  more:
<img width="513" alt="image" src="https://github.com/SoftFever/OrcaSlicer/assets/10420187/ab31fa8c-15ea-43d9-bb11-93a4304eae82">

The gcode window can be banned to hell by pressing [C].

If this helps and solves your hellish problem, please the hell close this issue.

_Please don't take this post too seriously - the hell._

**Comment by boromyr** [comment]
Is there a way to hide it using a keyboard shortcut?

**Reply by foreachthing** [reply]
pressing [C]...

**Reply by boromyr** [reply]
C is for the gcode panel, not for the statistics panel.

**Reply by foreachthing** [reply]
Yeah, you're right! 😄
Maybe there's a hint in [?]?

**Reply by boromyr** [reply]
I have searched, but nothing.

**Reply by foreachthing** [reply]
Yes, there is nothing - time for a FR.

**Comment by tavdog** [comment]
I too would like that window to not auto pop up every slice.

**Comment by dstasiuk** [comment]
I would also like to know.

**Comment by gixxerfool** [comment]
Does anyone know if there is a solution to this? It is really annoying the window covers up the part every time it gets sliced.  Default to collapsed would be a welcome feature.

**Comment by kaspesla** [comment]
please fix this to not always appear. bizarre that there's no way to ban it.

**Comment by andrewleek** [comment]
I have made a PR to fix this issue: https://github.com/SoftFever/OrcaSlicer/pull/8172

Note that I havent set it to default to collapsed, but I have set it so that if you collapse it, it wont reopen on every slice (basically the open/close state persists across slicing)

**Reply by cbenz** [reply]
Could it be persisted across app restart (saved in a config or state file maybe)? I could try a PR if the idea is OK
